include/chemfiles/files/NcFile.hpp

```diff
@@ -55,9 +55,14 @@ namespace nc {
 
         /// Get the attribute `name`.
         std::string attribute(const std::string& name) const;
```

Comment:
How about renaming this one to `string_attribute`, and the one below to `float_attribute`? (float_attribute feels more natural than attribute_float, and if the float one has the type in the name, the string one should too)

Reply:
That's a good idea. It's more descriptive and consistent.